configurator/frontend/src/ui/components/ConfigurableFieldsForm/ConfigurableFieldsForm.tsx

```diff
@@ -166,6 +168,10 @@ const ConfigurableFieldsForm = ({
       const fieldsValue = form.getFieldsValue();
 
       switch (type?.typeName) {
+        case 'description':
+          return (
+              <div style={{paddingTop: 5}} dangerouslySetInnerHTML={{__html: defaultValue}}/>
```

Comment:
For unification purposes, it is preferable to use tailwind classes instead of the inline styles. Consider `className="pt-2"` instead.

Also, I will attempt to get the same result without setting the html.

Reply:
Could find right solution. Left as is

Reply:
fixed in aae40a1